src/main/java/com/sigopt/model/Assignments.java

```diff
@@ -31,4 +31,17 @@ public Builder set(String key, Double value) {
             return this;
         }
     }
+
+    public String getCategorical(String name) {
+        return (String)this.get(name);
+    }
```

Comment:
maybe getString ? since it's more about the assignment value than the corresponding parameter type